policies/templates/gcp_cloudsql_ssl_v1.yaml

```diff
@@ -0,0 +1,98 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+apiVersion: templates.gatekeeper.sh/v1alpha1
+kind: ConstraintTemplate
+metadata:
+  name: gcp-cloudsql-ssl-v1
+  annotations:
+    # Example of tying a template to a CIS benchmark
+    benchmark: CIS11_5.03
+spec:
+  crd:
+    spec:
+      names:
+        kind: GCPCloudSQLSSLConstraintV1
+        plural: gcpcloudsqlsslconstraintsv1
+      validation:
+        openAPIV3Schema:
+          properties: {}
+  targets:
+   validation.gcp.forsetisecurity.org:
+      rego: |
+            #INLINE("validator/cloudsql_ssl.rego")
+
+            #
+
+            # Copyright 2018 Google LLC
+
+            #
+
+            # Licensed under the Apache License, Version 2.0 (the "License");
+
+            # you may not use this file except in compliance with the License.
+
+            # You may obtain a copy of the License at
+
+            #
+
+            #      http://www.apache.org/licenses/LICENSE-2.0
+
+            #
+
+            # Unless required by applicable law or agreed to in writing, software
+
+            # distributed under the License is distributed on an "AS IS" BASIS,
+
+            # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+
+            # See the License for the specific language governing permissions and
+
+            # limitations under the License.
+
+            #
+
+            
+
+            package templates.gcp.GCPCloudSQLSSLV1
+
+            
+
+            import data.validator.gcp.lib as lib
+
+            
+
+            deny[{
+
+            	"msg": message,
+
+            	"details": metadata,
+
+            }] {
+
+            	asset := input.asset
+
+            	asset.asset_type == "sqladmin.googleapis.com/Instance"
+
+            	asset.resource.settings.ipConfiguration.requireSsl == false
```

Comment:
My template file actually hardcoded this value... which should be provided by a constraint file.